### DISCUSSION TITLE
MAOIs + DXM

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#89] [Drug Interactions]
providing source

Gillman, P. K. (2005). Monoamine oxidase inhibitors, opioid analgesics and serotonin toxicity. BJA: British Journal of Anaesthesia, 95(4), 434–441. https://doi.org/10.1093/bja/aei210

**Comment by utaninja** [comment]
Adding to this -
Edinoff AN, Swinford CR, Odisho AS, Burroughs CR, Stark CW, Raslan WA, Cornett EM, Kaye AM, Kaye AD. Clinically Relevant Drug Interactions with Monoamine Oxidase Inhibitors. Health Psychol Res. 2022 Nov 3;10(4):39576. doi: 10.52965/001c.39576. PMID: 36425231; PMCID: PMC9680847.

Seems to be a concern of serotonin syndrome. I think also hypertension and possibly other stuff? I'm not so familiar with MAOIs not exactly sure.

**Reply by zelixir25** [reply]
the hypertension risk seems to only apply to other cold medicines like phenylephrine. for dxm specifically it seems to be just risk of SS/ST. the current classification/wording is fine i think

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/330
